src/components/NavBar/Link.tsx

```diff
@@ -23,15 +26,24 @@ const StyledLink = styled(NavLink)`
   }
 `;
 
-
 interface LinkProps {
- to: string;
- title: string;
- text: string;
-};
+  to: string;
+  title: string;
+  text: string;
+}
 
 const Link: React.FC<LinkProps> = (props) => (
-  <StyledLink to={props.to} title={props.title}>
+  <StyledLink
+    activeClassName="any"
+    isActive={(match) => {
```

Comment:
To simplify, instead of returning true and false, return the condition
e.g. return !(condition)